tool/RELEASE_INSTRUCTIONS.md

```diff
@@ -184,14 +184,26 @@ you may need to hard reload and clear your browser cache.
 
 If `package:devtools_app_shared`, `package:devtools_extensions`, or
 `package:devtools_shared` have unreleased changes, these packages may need to be
-published to pub.
+published to pub. Publish the backages whith versions they have, even if it is
+`.dev` versions.
 
 **Before publishing these packages, please message the DevTools Team chat room to ask if there
 are any reasons why we should wait.** Since these packages follow their own release schedules,
 it is possible that there are changes that are not ready to publish.
 
-From the respective `devtools/packages/devtools_*` directories, run:
+From the respective `devtools/packages/devtools_*` directories, run `flutter pub publish`. You can copy
+commands from here:
+
    ```shell
+   # Assuming you are in one of packages root.
+
+   cd ../devtools_app_shared
+   flutter pub publish
+
+   cd ../devtools_extensions
+   flutter pub publish
+
+   cd ../devtools_shared
```

Comment:
publish devtools_shared before devtools_app_shared, since devtools_shared is a dep of devtools_app_shared